### PR TITLE
PLF-8447 : Resume Daily and weekly digest mails jobs

### PR DESCRIPTION
**Pull request by rdenarie**
This commit add an upgrade plugin to resume jobs in environments in which the migration is already done.

### CODE REVIEW

File: extension/webapp/src/main/webapp/WEB-INF/conf/platform/upgrade/upgrade-configuration.xml [hunk 1]
```diff
@@ -609,5 +609,33 @@
...
+           </value-param>
+           <value-param>
+               <name>plugin.upgrade.execute.once</name>
+               <description>Execute this upgrade pluginonly once</description>
+               <value>true</value>
+           </value-param>
+           <value-param>
+               <name>plugin.upgrade.target.version</name>
+               <description>Target version of the plugin</description>
+               <value>5.3.0</value>
```
Comment (rdenarie) [inline extension/webapp/src/main/webapp/WEB-INF/conf/platform/upgrade/upgrade-configuration.xml]: I'm not sure about the version to put here.
In fact, this fix will be backported to plf 5.3.0 so for people migrating from 5.2.3 to 5.3.0, the UP will start correctly.
But, for example, on tribe, as the platform is already on 6.0.x, I'm not sure this up will start.
WDYT ?

Comment (boubaker) [reply extension/webapp/src/main/webapp/WEB-INF/conf/platform/upgrade/upgrade-configuration.xml]: you can put 5.2.0, the version on which the problem is reproduced.

Comment (rdenarie) [reply extension/webapp/src/main/webapp/WEB-INF/conf/platform/upgrade/upgrade-configuration.xml]: done

File: component/upgrade/plugins/src/main/java/org/exoplatform/platform/upgrade/plugins/ResumeDigestJobUpgradePlugin.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  }
+
+
+  @Override
+  public void processUpgrade(String oldVersion, String newVersion) {
+    try {
+      schedulerService.resumeJob("NotificationDailyJob", "Notification");
+      schedulerService.resumeJob("NotificationWeeklyJob", "Notification");
+    } catch (Exception e) {
+      e.printStackTrace();
```
[the remark this answers is not in the document]
Comment (rdenarie) [reply component/upgrade/plugins/src/main/java/org/exoplatform/platform/upgrade/plugins/ResumeDigestJobUpgradePlugin.java]: done

File: extension/webapp/src/main/webapp/WEB-INF/conf/platform/upgrade/upgrade-configuration.xml [hunk 3]
```diff
@@ -609,5 +609,33 @@
...
+           </value-param>
+           <value-param>
+               <name>plugin.upgrade.execute.once</name>
+               <description>Execute this upgrade pluginonly once</description>
+               <value>true</value>
+           </value-param>
+           <value-param>
+               <name>plugin.upgrade.target.version</name>
+               <description>Target version of the plugin</description>
+               <value>5.2.0</value>
```
Comment (boubaker) [inline extension/webapp/src/main/webapp/WEB-INF/conf/platform/upgrade/upgrade-configuration.xml]: Sorry, it's better to use "5.3.0" instead.
You can made a test from 5.2.1 to 5.2.3 and then to 5.3.0, the plugin will not be executed.

Comment (rdenarie) [reply extension/webapp/src/main/webapp/WEB-INF/conf/platform/upgrade/upgrade-configuration.xml]: done

**Pull request merged**